### PR TITLE
Companion for `substrate/pull/8903`

### PR DESCRIPTION
**Pull request by kianenigma**
https://github.com/paritytech/substrate/pull/8903

### CODE REVIEW

File: runtime/kusama/src/lib.rs [hunk 1]
```diff
@@ -378,6 +378,11 @@ impl pallet_election_provider_multi_phase::Config for Runtime {
 	type OnChainAccuracy = Perbill;
 	type Fallback = Fallback;
 	type BenchmarkingConfig = ();
+	type ForceOrigin = EnsureOneOf<
+		AccountId,
+		pallet_collective::EnsureProportionAtLeast<_2, _3, AccountId, TechnicalCollective>,
```
Comment (kianenigma) [inline runtime/kusama/src/lib.rs:383]:
```suggestion
		EnsureRoot<AccountId>,
```

File: runtime/polkadot/src/lib.rs [hunk 2]
```diff
@@ -359,6 +359,11 @@ impl pallet_election_provider_multi_phase::Config for Runtime {
 	type CompactSolution = NposCompactSolution16;
 	type Fallback = Fallback;
 	type BenchmarkingConfig = ();
+	type ForceOrigin = EnsureOneOf<
+		AccountId,
+		pallet_collective::EnsureProportionAtLeast<_2, _3, AccountId, TechnicalCollective>,
```
Comment (kianenigma) [inline runtime/polkadot/src/lib.rs:364]:
```suggestion
		EnsureRoot<AccountId>,
```

Comment (kianenigma) [reply runtime/polkadot/src/lib.rs:364]: maybe we want root here?

**Review by shawntabrizi — APPROVED** [review]
with root, makes sense to me

**Review by gavofyork — APPROVED** [review]

**Pull request merged**